app/main.py

```diff
@@ -1,8 +1,23 @@
 class Person:
-    # write your code here
-    pass
+
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = [Person(name=person["name"], age=person["age"])
+                   for person in people]
+    for person in people:
+        if person.get("husband"):
+            main_person = Person.people[person["name"]]
+            husband_person = Person.people[person["husband"]]
+            main_person.husband = husband_person
+        elif person.get("wife"):
+            main_person = Person.people[person["name"]]
+            wife_person = Person.people[person["wife"]]
+            main_person.wife = wife_person
```

Comment:
The `wife` attribute is not defined in the `Person` class. This can lead to a `AttributeError` if you try to access this attribute before it's set. It's better to define all attributes in the class constructor. This is a bug.